### PR TITLE
Integrated the RSS feeds into the GUI

### PR DESCRIPTION
**Pull request by egonw**
Making it look like (feedback welcome!), along with similar icons in /venue/ and /topic/:

image

### CODE REVIEW

File: scholia/app/views.py [hunk 1]
```diff
@@ -63,6 +63,10 @@ def register_converter(state):
...
...aAhyIPvQdNRw9sbAp77hsem1T5O1MpwIpJtT+Y5tjPE9yDMQ8tJw4GNIb/5jseLS3NE4iV45/SoxApWK5eMmdWbIU77cpTtT+SJJAQbVvglc8bPcXnlBz6ROBzZrpkyr7iu+hIH3wtwohZXDvxRc2xnnsADJyK7qzpjoGqEYvNqWwN0NELoWzrlRiXH39TUv50nyFsDOlEIA0hWKryM7jaqikhO74fNP/PxPQsSThSEEJSf45BpMgjZRbiMg9b1YKpLyIoJe4dFPgtaK4yBN14uxYHRVyg6A4mjBcffLK1TB56v8HMS6djUFuQFMgtBHrQu8kWpIFYuyHfa/dtOGYRQCAmpYQIdQGDAzxVYpLGFWsVg8GsFQQ7Q0P88gVSF84aggyKoJ1JWTmeLKaDwcYkRG2w5XODp+U7B1qdKNeBnYevjGqPtHBfJyb2lSiiPKzCC5uOlDLL8HEGAlW+MIMhLfE8Q5GS3PBMKyvor8ln7vvmEQCAwWhAtk/hZgZ+TNB4SnDkCZ47Cqf2WLucykljSyookJTovCDxJW52g4b3C3H3gdQj8TpsGbU0iMWKDg5YrELm7AOt6yoV1/6w5sSdg5iLF8TcMz38zwHEUOiimsCM11gzCgJuEaFLidUAuDQ0HDPlCQmg/DYLi2iBnKasOAK3QBqLl1p11IMhlIPBEz4oc37NPkBMl1Xq2FZyYrV90YA+W96wHdbZaZtglpLMZ4v0KzRej0GEE466wTdF/ii9BBY+AsDETiQmm/J1ASHBc2L/RcHq/QRRwTwcw9mpJvJ91O6lg93qDDg1OVDDmM4IwbzfLZ+HgFm37BAYiCZCO/dvPgVKgIhTCw84XPZOTtnJ6vhPCyojE7f10APnOYggj7HxjQAoQys7tijev3SV07hVrsiuKbfHb459H5p/BhDBqmsGJGjA2TpUyHKkpXspoGHaxQQhLbIQ0nNhdUFgEqkZaSm00hHlDW32xgaoiIGWPrrzpXTCFfu8ufNelheyhDGNlBLnSXqagMFcVMl73PoIwgEDdIh7LPktfPTi9KPZVk+o/zcsHZQgN2lgr5DsLAFRA7Fg5mKDI8oJcQfOF5kkXUHUdsudFPqyZbEzvPquUfX+nMfRWlpSlXS0gppy0CNQOuSb707PvfdnH9zOYGfXJp8BPxvvH/wGCg57vPihW9AAAAABJRU5ErkJggg=='
```
Comment (egonw) [inline scholia/app/views.py]: Forgot to add this in the comment, but this image is CCZero and originates from: https://pixabay.com/en/rss-news-feed-blog-web-icon-40674/

**Comment by fnielsen** [comment]
Would it not be better to have the image as a static file? At the moment `flake8` is complaining.

**Comment by egonw** [comment]
@fnielsen, where should I put the image?

**Comment by egonw** [comment]
@fnielsen, ok, patch updated. Can you please have another look?

**Pull request merged**